packages/runner/src/lib/FunctionScanTarget.ts

```diff
@@ -0,0 +1,25 @@
+import fastify from 'fastify';
+
+export class FunctionScanTarget {
+  private readonly server = fastify();
+
+  public async start<T>(
+    fn: (input: T) => Promise<unknown>
+  ): Promise<{ url: string }> {
+    this.server.post('/', async (request, reply) => {
```

Comment:
Extract the callback as a field with a arrow function for simplicity

Reply:
extracted in https://github.com/NeuraLegion/sectester-js/pull/212/commits/de9ab2b2d2c077e7fa0ff2f51bcbcff8080f8583